backend/geonature/core/gn_meta/routes.py

```diff
@@ -103,11 +104,17 @@ def get_datasets():
     :returns:  `list<TDatasets>`
     """
     params = MultiDict(request.args)
+    if request.is_json:
+        params.update(request.json)
     fields = params.get("fields", type=str, default=[])
     if fields:
         fields = fields.split(",")
     if "create" in params:
-        query = TDatasets.query.filter_by_creatable(params.pop("create"))
+        create = params.pop("create").split(".")
+        if len(create) > 1:
+            query = TDatasets.query.filter_by_creatable(create[0], object_code=create[1])
```

Comment:
Peut être nommé les paramètres ?
```suggestion
            query = TDatasets.query.filter_by_creatable(module_code=create[0], object_code=create[1])
```
Je rajouterais bien aussi  dans la docstring le fait qu'on peut passer le module_code + le object_code dans "create". ça fait une mini doc sur l'API pour qu'on s'en souviennt

Reply:
Pas faux, on risque d'oublier pourquoi ya ca sinon